radio/util/tts_it.py

```diff
@@ -14,40 +14,40 @@
 systemSounds.append(("mille", filename(PROMPT_SYSTEM_BASE + 102)))
 for i, s in enumerate(["virgola", "un", "e", "meno", "ora", "ore", "minuto", "minuti", "secondo", "secondi"]):
     systemSounds.append((s, filename(PROMPT_SYSTEM_BASE + 103 + i)))
-for i, (s, f) in enumerate([(u"volt", "volt0"),
-                            (u"ampere", "amp0"),
-                            (u"milliampers", "mamp0"),
+for i, (s, f) in enumerate([(u"volts", "volt0"),
+                            (u"ampèr?", "amp0"),
+                            (u"milliampèr?", "mamp0"),
                             (u"nodo", "knot0"), ("nodi", "knot1"),
                             (u"metro al secondo", "mps0"), ("metri al secondo", "msp1"),
                             (u"piede al secondo", "fps0"), ("piedi al secondo", "fps1"),
-                            (u"chilometro ora", "kph0"), ("kilometers per hour", "kph1"),
+                            (u"chilometro ora", "kph0"), ("chilometri orari", "kph1"),
                             (u"miglio per ora", "mph0"), ("miglia per ora", "mph1"),
                             (u"metro", "meter0"), ("metri", "meter1"),
                             (u"piede", "foot0"), ("piedi", "foot1"),
                             (u"grado celsius", "celsius0"), ("gradi celsius", "celsius1"),
                             (u"grado fahrenheit", "fahr0"), ("gradi fahrenheit", "fahr1"),
                             (u"percento", "percent0"), (u"percento", "percent1"),
-                            (u"milliampere ora", "mamph0"), (u"milliampere ora", "mamph1"),
+                            (u"milliampèr? ora", "mamph0"), (u"milliampèr? ora", "mamph1"),
```

Comment:
I completely trust you on the sound since I do not speak Italian at all. The issue is that those string are used for generation of nightlies sd card pack (which might be using different tools) and also are going into the cvs files, to be then used in even more different tools on various platform, so you fine tuned spelling for one specific engine is likely to not sound good on another one, or break another one (some don't like ? without space for example)

Next RC will have new sound generations for Italian